packages/connect-wallet-modal/src/components/WalletsModal/components/ConnectWalletModalLayout/ConnectWalletModalLayout.tsx

```diff
@@ -61,6 +63,17 @@ export const ConnectWalletModalLayout = ({
   }, [isShownOtherWallets, onInputClear]);
 
   useEffect(() => {
+    // Additional check because `@supports selector(::-webkit-scrollbar)`
+    // passes as true on iOS device, but styles will not really apply
+    // Also the `helpers/userAgents` does not work really well in my tested case,
+    // so there is another iPhone/iPad check implementation
```

Comment:
I don't like this solution because now we have several different `isIOS` checks.
If `user-agents` check is not working for some reason (not sure why), then it must be fixed or replaced.

Reply:
`user-agents` on ipad contains information about OSX, not ios for some reason